BUILD/task_order/Quantum Terrarium.dat

```diff
@@ -1,20 +1,11 @@
 LX_freeCombatsTask
 catBurglarHeist
 auto_breakfastCounterVisit
-chateauPainting
-LX_faxing
```

Comment:
these two will be useful once it is out of standard. no reason to remove them. they are harmless now and will be useful in a month or two